include/miopen/miopen.h

```diff
@@ -634,7 +634,7 @@ MIOPEN_EXPORT miopenStatus_t miopenCreateTensorDescriptor(miopenTensorDescriptor
 
 /*! @brief Set shape of 4D tensor
  *
- * Interface for setting 4-D packed tensor shape.
+ * Interface for setting 4-D tensor shape. MIOpen currently implements NCHW and NHWC layout.
```

Comment:
Could we please change the sentence to,

MIOpen currently implements both NHWC (Number of samples, Height, Channels, Width) and NCHW (Number of samples, Channels, Height, Width) layouts. 

@junliume 

Let me know if I have misunderstood the abbreviations. Thanks so much.

Reply:
@iq136boy could you follow up with the comment?

Reply:
@Rmalavally "Number of samples" (N) is actually "Batch size"